manifests/source/export.pp

```diff
@@ -26,8 +32,10 @@
 # Copyright 2015 CoverMyMeds
 #
 define sentry::source::export (
-  $language = 'Other',
-  $env      = undef,
+  $language     = 'Other',
+  $organization = $::sentry::organization,
+  $team         = $::sentry::team,
```

Comment:
Same for this parameter. Why check existence below if you're functionally doing the same thing?

If undef is passed to the definition in this parameter, it'll use the default set here.